cas-server-core-api-authentication/src/main/java/org/jasig/cas/authentication/RootCasException.java

```diff
@@ -12,7 +12,7 @@
  * see org.jasig.cas.authentication.handler.AuthenticationException
  * @since 4.0.0
  */
-public abstract class RootCasException extends Exception {
+public abstract class RootCasException extends RuntimeException {
```

Comment:
do we really want to do this? This relaxes all the `AbstractTicketException` checked exceptions.